audiofile/core/convert.py

```diff
@@ -16,6 +14,11 @@
 logging.getLogger('sox').setLevel(logging.CRITICAL)
 
 
+# Import sox after disabling warning
+# as it is ok to not have the binary present
+import sox  # pragma: E402
```

Comment:
Moving the import os `sox` after we disable the sox warnings should solve the issue of showing the warning (and failing building docs when `sox` is not installed). The problem is that I get a flake8 E402 error here even though I use `pragma`, or do I something wrong?